google/cloud/internal/rest_client.cc

```diff
@@ -48,9 +48,16 @@ Status MakeRequestWithPayload(
       concatenated_payload += std::string(p.begin(), p.end());
     }
     encoded_payload = impl.MakeEscapedString(concatenated_payload);
+    impl.SetHeader(absl::StrCat("content-length: ", encoded_payload.size()));
     return impl.MakeRequest(http_method,
                             {{encoded_payload.data(), encoded_payload.size()}});
   }
+
+  int content_length = 0;
```

Comment:
This should be a `std::size_t`, on MSVC `int` is too narrow (in theory, in practice it is fine, but the compiler will warn).

Reply:
Ah, you changed that already. Ignore this.